superset-frontend/plugins/legacy-preset-chart-deckgl/src/utilities/Shared_DeckGL.jsx

```diff
@@ -39,7 +39,7 @@ const DEFAULT_VIEWPORT = {
 
 const sandboxUrl =
   'https://github.com/apache/incubator-superset/' +
```

Comment:
is this one still ok ^^^?

Reply:
It works, but it's worth changing. Nice catch.